rootless.md

```diff
@@ -0,0 +1,54 @@
+# Shortcomings of RootLess Podman
+
+* Not a ton of “How To” documentation or documentation in general
+* If you don’t have /etc/subuid and /etc/subgid setup for a user, then it can easily fail
+* This causes issues on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
```

Comment:
Does "This" refer to subuid/subgid setup or rootless in general?

Reply:
It is referring to the /etc/subuid and /etc/subgid issue.